style.css

```diff
@@ -1,6 +1,7 @@
 p {
   color: red;
   font: fira-code;
+  font-size: 2em;
```

Comment:
this font size might be too big

Reply:
Okay, will change it to something smaller